client/platform/web-girder/views/AnnotationTags.vue

```diff
@@ -0,0 +1,89 @@
+<script lang="ts">
+import {
+  defineComponent, ref,
+} from '@vue/composition-api';
+import {
+  useAnnotationTag,
+  useAnnotationTags,
+  useHandler,
+} from 'vue-media-annotator/provides';
+
+export default defineComponent({
+  name: 'AnnotationTags',
+  description: 'Annotation Tags',
+
+  setup() {
+    const currentTag = useAnnotationTag();
+    const tags = useAnnotationTags();
+    const newTag = ref('');
+    const validForm = ref(false);
+    const { save, tagChange } = useHandler();
+    const selectedTag = (tag: string) => {
+      tagChange(tag);
+    };
+
+    const addTag = async () => {
+      await save(newTag.value);
+      tagChange(newTag.value);
+    };
+
+    return {
+      currentTag,
+      selectedTag,
+      tags,
+      newTag,
+      addTag,
+      validForm,
+    };
+  },
+});
+</script>
+
+<template>
+  <div>
+    <p> Available Tags</p>
+    <v-list>
+      <v-list-item
+        v-for="tag in tags"
+        :key="tag"
+      >
+        <v-chip
+          outlined
+          :color="tag === currentTag || (!currentTag && tag === 'default') ? 'cyan' : 'white'"
+          @click="selectedTag(tag)"
+        >
+          {{ tag }}
+        </v-chip>
+      </v-list-item>
+    </v-list>
+    <v-divider />
+    <v-expansion-panels>
+      <v-expansion-panel>
+        <v-expansion-panel-header> Add New Tag</v-expansion-panel-header>
+        <v-expansion-panel-content>
+          <p>
+            Add a new Tag with a custom name that can be used to reference it in the future.
+            'default' is a reserved tag which can't be used.
+            Adding a new tag will tag the current annotations and copy them to the new tag.
+          </p>
```

Comment:
```suggestion
          <p>
          Enter a name for a new annotation group. Grouping annotations allows for easy comparison. Creating a new 
          group will add all current annotations to that group.
          </p>
```

I do think we need to change the name from tag to group or collection or something like that. Currently new annotations get added to the selected tag as well. Should this be included here or is that intuitive enough?

Reply:
Can't use group, we already have that reserved for groupings of tracks.  Collection has a meaning in a girder context.

Reply:
batch?

Reply:
If we're going towards slicer CLI in the future, the girder plugin has terminology using 'batch' for running the same job on multiple datasets/items.  I don't know if that concept overlaps completely with what is done here to use a similar name or if they are two different and should use different names to prevent confusion.
https://github.com/girder/slicer_cli_web/blob/2cfbb92256f840e1edd3004632432ce1d0634209/README.rst#batch-processing

Reply:
Discussion offline concluded in using 'Annotation-Set'